plugin.cmake

```diff
@@ -14,6 +14,11 @@
 #  limitations under the License.
 ###############################################################################
 
+# support the "if(TEST ...)" command
+if(POLICY CMP0064)
```

Comment:
I'm not personally concerned about it, but I could address it.  One issue is that I can't think of a blanket way to improve the error message.  For this specific case, we could check a combination of cmake version + js testing option, but there are other possible reasons why a test would not be created by `add_web_client_test` or `add_python_test`.

Perhaps the best option is to add a new argument to our cmake functions to add custom environment variables.